dataprofiler/validators/base_validators.py

```diff
@@ -20,7 +20,7 @@ def is_in_range(x: Union[float, int], config: dict) -> bool:
     :returns: bool
     """
     try:
-        return config["start"] <= float(x) <= config["end"]
+        return float(config["start"]) <= float(x) <= float(config["end"])
```

Comment:
is this conversion needed? interesting

Reply:
I ended up needing to do this because mypy doesn't know if `config["start"]` and `config["end"]` actually exist so it assigns them the type `Any`. Expressions involving `Any` evaluate to type `Any` which we're not allowed to return due to our new mypy flags. 

I could also use `cast` to prevent the use of a runtime cast but I figured since it's already in a try except block, any issues would be handled the same way as the cast `float(x)`.